make requirements fail

* edx-salesforce version: latest
* Django version: 
* Python version: 2.7
* Operating System: Linux vagrant 4.4.0-31-generic #50-Ubuntu SMP Wed Jul 13 00:07:12 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

### Description
make requirements fails

Side note, I found the documents via google search, and saying 'make requirements' .. doesnt help that much. I had to find the repo, and assume I should pull the code, and put it within the edx-platform directory. In other words, it would be nice to have just a little more hand holding within the documentation.

### What I Did
Following the instructions on the wiki: https://edx-salesforce.readthedocs.io/en/latest/

```
$ make requirements
pip install -qr requirements.txt --exists-action w
Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'
Makefile:36: recipe for target 'requirements' failed
make: *** [requirements] Error 1
```

There is no requirements.txt file in the repo.
What do I need to do, in order to get edx-salesforce to 'make'

Thanks for any help.

--- a/Makefile
+++ b/Makefile
@@ -33,7 +33,7 @@ local-requirements:
 	pip install -qr requirements/local.txt --exists-action w
 
 requirements:
-	pip install -qr requirements.txt --exists-action w
+	pip install -qr requirements/production.txt --exists-action w
 
 test: clean
 	coverage run ./manage.py test edx_salesforce --settings=settings.test

